### PR TITLE
GDPR-152: Update project structure to include integrations tests & update build

### PR DESCRIPTION
**Pull request by hyderkhanmoj**

### CODE REVIEW

File: src/testIntegration/java/uk/gov/justice/hmpps/datacompliance/services/health/HealthCheckIntegrationTest.java [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 public class HealthCheckIntegrationTest extends IntegrationTest {
 
-    @Test
+   // @Test
+    //FIXME: UNDO THIS COMMENT
```
Comment (brightonsbox) [inline src/testIntegration/java/uk/gov/justice/hmpps/datacompliance/services/health/HealthCheckIntegrationTest.java:12]: Is this something that you're doing as a follow up?

File: src/testIntegration/resources/application-test.yml [hunk 2]
```diff
@@ -9,7 +9,32 @@ pathfinder.api.base.url: http://localhost:8997
...
+  sqs:
+    provider: embedded-localstack
+    queue.name: data_compliance_request_queue
+    queue.url: http://localhost:4576/queue/data_compliance_request_queue
+    dlq.name: data_compliance_request_dead_letter_queue
+    endpoint.url: http://localhost:4576
+
+data.compliance.response:
+  sqs:
+    provider: localstack
```
Comment (brightonsbox) [inline src/testIntegration/resources/application-test.yml:23]: Why is there a mix of providers for request / response here?

**Review by brightonsbox — DISMISSED** [review]
LGTM! This looks great!

File: src/testIntegration/resources/application-test.yml (hunk 2 above)
Comment (hyderkhanmoj) [reply src/testIntegration/resources/application-test.yml:23]: This is mistake, thanks for spotting it. It should be 'embedded-localstack'

File: src/testIntegration/java/uk/gov/justice/hmpps/datacompliance/services/health/HealthCheckIntegrationTest.java (hunk 1 above)
Comment (hyderkhanmoj) [reply src/testIntegration/java/uk/gov/justice/hmpps/datacompliance/services/health/HealthCheckIntegrationTest.java:12]: yes its something that will be fixed in the next commit

**Review by brightonsbox — APPROVED** [review]
LGTM

**Pull request merged**